dev/unix/boot-install.sh

```diff
@@ -1,76 +1,94 @@
 #!/usr/bin/env bash
 
-# This is the bootstrap Unix installer served by `https://get.notionjs.com`.
+# This is the bootstrap Unix installer served by `https://get.volta.sh`.
 # Its responsibility is to query the system to determine what OS (and in the
 # case of Linux, what OpenSSL version) the system has, and then proceed to
-# fetch and install the appropriate build of Notion.
+# fetch and install the appropriate build of Volta.
 
-notion_get_latest_release() {
-  curl --silent https://www.notionjs.com/latest-version
+volta_get_latest_release() {
+  curl --silent https://www.volta.sh/latest-version
 }
 
-notion_eprintf() {
+volta_eprintf() {
   command printf "$1\n" 1>&2
 }
 
-notion_info() {
+volta_info() {
   local ACTION
   local DETAILS
   ACTION="$1"
   DETAILS="$2"
   command printf '\033[1;32m%12s\033[0m %s\n' "${ACTION}" "${DETAILS}" 1>&2
 }
 
-notion_error() {
+volta_error() {
   command printf '\033[1;31mError\033[0m: ' 1>&2
-  notion_eprintf "$1"
+  volta_eprintf "$1"
 }
 
-notion_warning() {
+volta_warning() {
   command printf '\033[1;33mWarning\033[0m: ' 1>&2
-  notion_eprintf "$1"
-  notion_eprintf ''
+  volta_eprintf "$1"
+  volta_eprintf ''
 }
 
-notion_request() {
+volta_request() {
   command printf "\033[1m$1\033[0m" 1>&2
-  notion_eprintf ''
+  volta_eprintf ''
 }
 
-notion_install_dir() {
-  printf %s "${NOTION_HOME:-"$HOME/.notion"}"
+legacy_install_dir() {
+  printf "%s" "${NOTION_HOME:-"$HOME/.notion"}"
+}
+
+# Check for a legacy installation from when the tool was named Notion.
+volta_check_legacy_installation() {
+  local LEGACY_INSTALL_DIR="$(legacy_install_dir)"
+  if [[ -n "$LEGACY_INSTALL_DIR" ]]; then
+      volta_eprintf ""
+      volta_error "You have existing Notion install, which can't be automatically upgraded to Volta."
+      volta_request "       Please delete $(legacy_install_dir) and try again."
```

Comment:
Nitpick: since there's already a variable assigned for this

```suggestion
      volta_request "       Please delete $LEGACY_INSTALL_DIR and try again."
```

Reply:
Considered that; mostly kept it the same for congruency with the other piece – but I'll update both instead.

Reply:
Done in ae84bae1!